google-cloud-storage/lib/google/cloud/storage/bucket/lifecycle.rb

```diff
@@ -105,6 +105,20 @@ def changed?
           #   only the date part (for instance, "2013-01-15"). This condition is
           #   satisfied when a file is created before midnight of the specified
           #   date in UTC.
+          # @param [String,Date] custom_time_before A date in RFC 3339 format with
+          #   only the date part (for instance, "2013-01-15"). This condition is
+          #   satisfied when the custom time on an object is before this date in UTC.
+          # @param [Integer] days_since_custom_time Represents the number of
+          #   days elapsed since the user-specified timestamp set on an object.
+          #   The condition is satisfied if the days elapsed is at least this
+          #   number. If no custom timestamp is specified on an object, the
+          #   condition does not apply.
+          # @param [Integer] days_since_noncurrent_time Represents the number of
+          #   days elapsed since the noncurrent timestamp of an object. The
+          #   condition is satisfied if the days elapsed is at least this number.
+          #   The value of the field must be a nonnegative integer. If it's zero,
+          #   the object version will become eligible for Lifecycle action as
+          #   soon as it becomes noncurrent.
```

Comment:
Missing `Relevant only for version enabled buckets`.

Reply:
Done.